cmd/core/main.go

```diff
@@ -205,12 +206,19 @@ func main() {
 		os.Exit(1)
 	}
 
+	plainValidator, err := validators.NewPlainValidator()
```

Comment:
IMO, anything specific to the plain provisioner should go in the `internal/proivisioner/plain` tree somewhere.